language/diem-framework/modules/0L/Oracle.move

```diff
@@ -266,6 +267,26 @@ address 0x1 {
           total_weight: 0,
         };
       }
+
+      /// This function will clear ballots if the current voting round has expired 
+      /// Unlike enter_new_upgrade_round, it will not start a new round 
+      /// it also does not clear the consensus so that it can be checked by the vm later
+      /// (that only happens if someone submits a proposal)
+      public fun clear_expired_ballots(vm: &signer) acquires Oracles {
+        Roles::assert_diem_root(vm);
+        let current_height = BlockHeight::get_height();
+        let upgrade_oracle = &mut borrow_global_mut<Oracles>(CoreAddresses::DIEM_ROOT_ADDRESS()).upgrade;
```

Comment:
I think we should use borrow_global here

Reply:
We can use `use 0x1::CoreAddresses::DIEM_ROOT_ADDRESS;` at the top of the file to avoid long lines here and some other places.